exer_11.py

```diff
@@ -0,0 +1,26 @@
+GP = [3, 6, 9, 12, 16, 18]
+AP =[2, 5, 8, 11, 15, 17]
+
+def checkAP(list):
+    a = list[0]
```

Comment:
Don't use `list` as a variable, it is reserved keyword. So please rename it to something else.